ui/src/kapacitor/containers/TickscriptPage.js

```diff
@@ -96,13 +163,18 @@ class TickscriptPage extends Component {
     this.setState({task: {...this.state.task, id: e.target.value}})
   }
 
+  HandleToggleLogsVisbility = () => {
+    this.setState({areLogsVisible: !this.state.areLogsVisible})
```

Comment:
Very minor thing here, but this should be lowercase `handleToggleLogsVisibility`

Reply:
👍